Source/Core/DolphinQt2/MenuBar.cpp

```diff
@@ -53,6 +55,27 @@
 
 #include "UICommon/GameFile.h"
 
+#if defined(_WIN32)
+bool FFmpeg_Win32_Exists()
+{
+  const std::string avcodec = File::GetExeDirectory() + DIR_SEP_WIN32 + "avcodec-58.dll";
+  const std::string avformat = File::GetExeDirectory() + DIR_SEP_WIN32 + "avformat-58.dll";
+  const std::string avutil = File::GetExeDirectory() + DIR_SEP_WIN32 + "avutil-56.dll";
+  const std::string swresample = File::GetExeDirectory() + DIR_SEP_WIN32 + "swresample-3.dll";
+  const std::string swscale = File::GetExeDirectory() + DIR_SEP_WIN32 + "swscale-5.dll";
+
+  if (File::Exists(avcodec) && File::Exists(avformat) && File::Exists(avutil) &&
```

Comment:
This can just be compressed into:

```cpp
return File::Exists(avcodec) && File::Exists(avformat) && File::Exists(avutil) &&
       File::Exists(swresample) && File::Exists(swscale);
```